packages/waggle/src/storage/storage.ts

```diff
@@ -294,37 +291,6 @@ export class Storage {
     console.timeEnd('initAllConversations')
   }
 
-  private getChannelsResponse(): ChannelInfoResponse {
-    const channels: ChannelInfoResponse = {}
-    for (const channel of Object.values(this.channels.all)) {
-      if (channel.keys) {
-        // TODO: create proper validators
-        channels[channel.name] = {
-          address: channel.address,
-          description: channel.description,
-          owner: channel.owner,
-          timestamp: channel.timestamp,
-          name: channel.name
-        }
-      }
-    }
-    return channels
-  }
-
-  public async updateChannels(): Promise<ChannelInfoResponse> {
-    /** Update list of available public channels */
-    if (!this.publicChannelsEventsAttached) {
-      this.channels.events.on('replicated', () => {
-        loadAllPublicChannels(this.io, this.getChannelsResponse())
-      })
-      this.channels.events.on('ready', () => {
-        loadAllPublicChannels(this.io, this.getChannelsResponse())
-      })
-      this.publicChannelsEventsAttached = true
-    }
-    return this.getChannelsResponse()
-  }
-
   protected getAllEventLogEntries(db: EventStore<any>): any[] {
     // TODO: fix typing
```

Comment:
how important are these items?

Reply:
I don't want to sound rude but there's a huge codebase in the project and I just don't know anything about the code that I didn't touch and this one is definitely something I see for the first time

Reply:
started slack thread to learn what the best practice for stuff like this is